core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyExtraTest.java

```diff
@@ -76,14 +73,18 @@ private int expectedAssociationNumber() {
 			// In Neo4j relationships are bidirectional
 			return 1;
 		}
+		else if ( TestHelper.getCurrentDialectType().equals( GridDialectType.ORIENTDB ) ) {
+			// In OrientDB ManyToMany relationships is like in RDBMS
+			return 3;
```

Comment:
Because:
math.getStudents().add( john );
math.getStudents().add( mario );
english.getStudents().add( kate );
In this way ... 3 rows in association table.